packages/@aws-cdk/aws-certificatemanager/lib/certificate.ts

```diff
@@ -40,33 +41,120 @@ export interface CertificateProps {
    * Has to be a superdomain of the requested domain.
    *
    * @default - Apex domain is used for every domain that's not overridden.
+   * @deprecated use `validation` instead.
    */
   readonly validationDomains?: {[domainName: string]: string};
 
   /**
    * Validation method used to assert domain ownership
    *
    * @default ValidationMethod.EMAIL
+   * @deprecated use `validation` instead.
    */
   readonly validationMethod?: ValidationMethod;
+
+  /**
+   * How to validate this certifcate
+   *
+   * @default CertificateValidation.fromEmail()
+   */
+  readonly validation?: CertificateValidation;
+}
+
+/**
+ * Properties for certificate validation
+ */
+export interface CertificationValidationProps {
+  /**
+   * Validation method
+   *
+   * @default ValidationMethod.EMAIL
+   */
+  readonly method?: ValidationMethod;
+
+  /**
+   * Hosted zone to use for DNS validation
+   *
+   * @default - use email validation
+   */
+  readonly hostedZone?: route53.IHostedZone;
+
+  /**
+   * A map of hosted zones to use for DNS validation
+   *
+   * @default - use `hostedZone`
+   */
+  readonly hostedZones?: { [domainName: string]: route53.IHostedZone };
+
+  /**
+   * Validation domains to use for email validation
+   *
+   * @default - Apex domain
+   */
+  readonly validationDomains?: { [domainName: string]: string };
+}
+
+/**
+ * How to validate a certificate
+ */
+export class CertificateValidation {
+  /**
+   * Validate the certifcate with DNS
+   *
+   * IMPORTANT: If neither `hostedZone` nor `hostedZones` is specified, DNS records
+   * must be added manually and the stack will not complete creating until the
+   * records are added.
+   *
+   * @param hostedZone the default hosted zone to use for all domains in the certificate
+   * @param hostedZones a map of hosted zones to use for domains in the certificate
+   */
+  public static fromDns(hostedZone?: route53.IHostedZone, hostedZones?: { [domainName: string]: route53.IHostedZone }) {
```

Comment:
I'm trying to think of a better way to do this interface. As-is, it's not clear how to use it -- if I need to provide the `hostedZones` map, do I also need to provide the `hostedZone`? If so, does it matter which is the "main" zone and which appears in the map? One other thought -- domain names to hosted zones are N:1; you may have 10 domain names with two top-level hosted zones; flipping the map around may be slightly more usable for some customers (I don't have the data to back that up off-hand though).

Maybe breaking this into two would alleviate some of the problems:
```
public static fromDns(hostedZone?: route53.IHostedZone) {...}
public static fromDnsMultiZone(hostedZoneMap: { [domainName: string]: route53.IHostedZone) {...}
```

Open to better naming and disagreements.